src/client/src/containers/Footer/index.tsx

```diff
@@ -268,6 +279,9 @@ const mapDispatchToProps = (
   },
   openPostGenModal: () => {
     dispatch(openPostGenModalAction());
+  },
+  updateCreateProjectButton: () => {
```

Comment:
NIT: Maybe `enableCreateProjectButton` would be a better name here as we don't pass any argument and imply true in this function

Reply:
I changed it to accept a boolean argument to update the state so that we can use the same action now to make it `false` as well. :)